src/sentry/static/sentry/app/utils/advancedAnalytics.tsx

```diff
@@ -0,0 +1,89 @@
+import * as qs from 'query-string';
+import {IntegrationsEventParams} from 'src/sentry/static/sentry/app/utils/integrationUtil';
+
+import {Organization} from 'app/types';
+import {trackAnalyticsEvent} from 'app/utils/analytics';
+import {uniqueId} from 'app/utils/guid';
+
+const ANALYTICS_SESSION = 'ANALYTICS_SESSION' as const;
+
+export const startAnalyticsSession = () => {
+  const sessionId = uniqueId();
+  window.sessionStorage.setItem(ANALYTICS_SESSION, sessionId);
+  return sessionId;
+};
+
+export const clearAnalyticsSession = () => {
```

Comment:
What happens if there's more than one analytics session

Reply:
hmmm we just assume only one. It just depends on what the active funnel is. I don't know of a use case where'd have overlapping funnels.